robottelo/performance/candlepin.py

```diff
@@ -25,9 +24,9 @@ class Candlepin(object):
     """
 
     # parameters for Candlepin Authentication
-    username = conf.properties['foreman.admin.username']
-    password = conf.properties['foreman.admin.password']
-    serverhost = conf.properties['main.server.hostname']
+    username = settings.server.admin_username
```

Comment:
This same issue exists in other places too, right? For example, `robottelo.test.CLITestCase` sets `cls.katello_user`, `cls.katello_passwd` and more:

```python
    @classmethod
    def setUpClass(cls):  # noqa
        """Make sure that we only read configuration values once."""
        super(CLITestCase, cls).setUpClass()
        cls.hostname = conf.properties['main.server.hostname']
        cls.katello_user = conf.properties['foreman.admin.username']
        cls.katello_passwd = conf.properties['foreman.admin.password']
        cls.key_filename = conf.properties['main.server.ssh.key_private']
        cls.root = conf.properties['main.server.ssh.username']
        cls.locale = conf.properties['main.locale']
        cls.verbosity = int(conf.properties['main.verbosity'])
```